test/e2e/scenario_0/scenario_0.cpp

```diff
@@ -0,0 +1,116 @@
+#include <gtest/gtest.h>
+
+#include <string>
+#include <util/pop_test_fixture.hpp>
+#include <vector>
+#include <veriblock/altintegration.hpp>
+#include <veriblock/config.hpp>
+#include <veriblock/mock_miner.hpp>
+
+using namespace altintegration;
+
+namespace generated {
+extern const std::string atv;
+extern const std::string vtbs;
+extern const std::string btcbootstraps;
+extern const std::string vbkbootstraps;
+}  // namespace generated
+
+static std::string dummy(std::string s) { return s; }
+
+template <typename T>
+static T fromHex(const std::string& h) {
+  return T::fromHex(h);
+}
+
+template <typename T>
+static std::vector<T> parse(const std::string& in,
+                            std::function<T(std::string)> transform = dummy) {
+  std::vector<T> ret;
+  std::istringstream s(in);
+  std::string line;
+  while (std::getline(s, line)) {
+    if (ParseHex(line).empty()) {
+      break;
+    }
+    ret.push_back(transform(line));
+  }
+  return ret;
+}
+
+struct Scenario0 : public ::testing::Test {
```

Comment:
Why is it Scenario0? Is it some basic, very important scenario to make it before scenario1?

Reply:
Yeah, it contains real data from vBTC :)